compiler/src/iree/compiler/Dialect/Stream/Transforms/Passes.cpp

```diff
@@ -29,7 +29,7 @@ static llvm::cl::opt<bool> clSpecializeEncodings(
         "Enables SpecializeEncodingPass in Stream pass pipeline. This pass is "
         "currently under development, so it is not enabled by default. It can "
         "only handle limited cases at this moment."),
```

Comment:
Shouldn't the description be updated to indicate that this is enabled by default now?

Reply:
Good catch, sorry for missing it! I think I'm going to remove the flag soon (e.g., by the end of the next week), it looks like the pass is stable enough to me.

Reply:
Here is the patch: https://github.com/iree-org/iree/pull/20279